raiden/network/transport/matrix.py

```diff
@@ -403,6 +412,10 @@ def stop(self):
         # parent may want to call get() after stop(), to ensure _run errors are re-raised
         # we don't call it here to avoid deadlock when self crashes and calls stop() on finally
 
+    def _post_sync(self, next_batch):
```

Comment:
Add a docstring here explaining why we got this `_post_sync()` function. 3 months down the line we will probably forget. Also add a type for `next_batch`.